src/context.rs

```diff
@@ -96,19 +83,22 @@ impl<'a> Context<'a> {
 
     /// Create a new module
     ///
-    /// Will return `None` if the module is disabled by configuration, by setting
-    /// the `disabled` key to `true` in the configuration for that module.
-    pub fn new_module(&self, name: &str) -> Option<Module> {
+    /// Can always return a new module as if a module was disabled it won't get
```

Comment:
Nitpick: I would leave `///` doc comments above functions for usage information, rather than contextual information. I think we can assume that a new module can always be returned from the function signature.